hdfs-lzo-text-scanner.h

```diff
@@ -149,6 +149,9 @@ class HdfsLzoTextScanner : public HdfsTextScanner {
   LzoFileHeader* header_;
 
   // Fills the byte buffer by reading and decompressing blocks.
+  // If applicable, attaches decompression buffers from previous calls that might still
+  // be referenced by returned batches to 'pool'. If 'pool' is nullptr the buffers are
+  // freed instead.
```

Comment:
Does the comment of ReadAndCompressData() below need to be updated too ?

Reply:
Done